projects/dsp-ui/src/lib/action/components/confirmation-dialog/confirmation-message/confirmation-message.component.spec.ts

```diff
@@ -0,0 +1,86 @@
+import { Component, OnInit } from '@angular/core';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+import { ReadIntValue } from '@dasch-swiss/dsp-js';
+import { ConfirmationMessageComponent } from './confirmation-message.component';
+
+/**
+ * Test host component to simulate parent component with a confirmation dialog.
+ */
+@Component({
+    template: `<dsp-confirmation-message [value]="testValue"></dsp-confirmation-message>`
+})
+class ConfirmationMessageTestHostComponent implements OnInit {
+
+    testValue: ReadIntValue;
+
+    constructor() {
+    }
+
+    ngOnInit() {
+        this.testValue = new ReadIntValue();
```

Comment:
Could you use the mocked int value from dsp-js-lib here?

Reply:
fixed in d9cde94